### ISSUE TITLE
Debug Keystore missing in Facebook Unity SDK

### ISSUE THREAD

**Issue by dps999** [facebook/facebook-sdk-for-unity#138]
Debug Keystore missing in Facebook Unity SDK
But also i haven't FacebookAndroidUtils.cs to fix path
What can i do? Thank you

**Comment by sarebots** [comment]
I'm facing the same issue. Did you find a fix?

**Comment by eddimull** [comment]
If that doesn't work, make sure that some of the other system variables are reading as expected (for Windows)

On the domain that my computer is connected to, the %HOMEDRIVE% variable was **not** set to C, so no matter how many times I ran variations of the above steps, it did not work. I wasn't able to overwrite the HOMEDRIVE system variable, so I had to make a copy of the .android content onto my z: drive. 

Long story short, check to make sure that %HOMEDRIVE%\%HOMEPATH%\.android\debug.keystore exists

**Comment by Miigga** [comment]
> I'm facing the same issue. Did you find a fix?
> 
> EDIT:
> 
> I fixed it, here are the steps:
> 
> * Move Android SDK to C:\Users<username>\AppData\Local\Android\sdk
> * Copy the entire contents of '.android' folder (including debug.keystore) from SDK to C:\Users<username>.android
> * Update the user environment variable for ANDROID_SDK_HOME
> * Update the SDK location in Android studio and Unity Preferences

Could you provide more details. Because I'm little lost
